docs/DeveloperGuide.md

```diff
@@ -1318,6 +1318,195 @@ testers are expected to do more *exploratory* testing.
+       order are shown in the result display box.
+    1. Test case (both client and recipe index used): `add-o c/1 r/1 op/4 od/12-12-2021 1200 oq/1`
+       <br> Expected: A new order is added to order list with the given details. Details of the added
+       order are shown in the result display box.
+    1. Test case: `add-o cn/Alex Yeoh cp/87438807 ca/Blk 30 Geylang Street 29. #06-40 rn/Chicken Rice ri/Rice-200-g, Chicken-1-whole op/4 od/12-12-2021 1200`
+       <br> Expected: A new order is added to order list with the given details. Details of the added
+       order are shown in the result display box.
+    1. Test case: `add-o cn/Alex Yeoh cp/87438807 ca/Blk 30 Geylang Street 29. #06-40 rn/Milo op/1.40 od/12-12-2021 1200`
+       <br> Expected: A new order is added to order list with the given details. Details of the added
+       order are shown in the result display box.
+    1. Test case: `add-o cn/Alex Yeoh cp/87438807 ca/Blk 30 Geylang Street 29. #06-40 rn/Nasi Lemak op/1.40`
+       <br> Expected: No order is added. Error details shown in the result display box.
+    1. Test case (all client and recipe details specified): `add-o cn/Alex Yeoh cp/847521698521145366247893451125 ca/Blk 30 Geylang Street 29. #06-40 rn/Chicken Rice ri/Rice-200-g, Chicken-1-whole op/4 od/12-12-2021 1200 oq/1`
+       <br> Expected: No order is added. Error details shown in the result display box.
+    1. Test case: `add-o c/0 r/1 op/4 od/12-12-2021 1200 oq/1`
+       <br> Expected: No order is added. Error details shown in the result display box.
+    1. Test case: `add-o op/4 od/12-12-2021 1200 oq/1`
+       <br> Expected: No order is added. Error details shown in the result display box.
+    1. Test case: `add-o`
+       <br> Expected: No order is added. Error details shown in the result display box.
+
+1. Adding an order ingredient
+    1. Prerequisites: Order list shows at least 1 order and at most 3 orders.
+    1. Test case: `add-oi 1 in/Rice iq/400 iu/g`
+       <br> Expected: The new ingredient is added to the first order.
+       Details of the new ingredient and the edited order are shown in the result display box.
+    1. Test case: `add-oi 4 in/Rice iq/400 iu/g`
+       <br> Expected: No order is edited. Error details shown in the result display box.
+    1. Test case: `add-oi 1 in/Chicken iq/0 iu/whole`
+       <br> Expected: No order is edited. Error details shown in the result display box.
+    1. Test case: `add-oi 1 in/Chicken iq/500000 iu/whole`
+       <br> Expected: No order is edited. Error details shown in the result display box.
+    1. Test case: `add-oi 1 in/Chi$ken iq/4 iu/whole`
+       <br> Expected: No order is edited. Error details shown in the result display box.
+    1. Test case: `add-oi 1 in/Chicken iq/4 iu/who%le`
+       <br> Expected: No order is edited. Error details shown in the result display box.
+    1. Test case: `add-oi in/Rice iq/400 iu/g 1`
```

Comment:
Remove this test case, it is the same as the one in line 1390

Reply:
Its different